libobs/media-io/video-io.c

```diff
@@ -533,14 +544,10 @@ void video_output_stop(video_t *video)
 	if (!video)
 		return;
 
-	if (video->initialized) {
-		video->initialized = false;
+	if (video->initialized && !video->stop) {
```

Comment:
You mentioned there were race conditions - do we need to be concerned with thread safety here, for example two threads calling `video_output_stop` on the same video output?

Reply:
There is no indication that we have that occuring, but we could add another mutex here to avoid that in the future.